### ISSUE TITLE
Scannet_preprocessed_data

### ISSUE THREAD

**Issue by zaiweizhang** [charlesq34/pointnet2#77]
Hi, 

For the scannet preprocessed data, are you using scannet v1 or v2?

Thanks!

**Comment by Bierfass** [comment]
They are using scannet v1. 
See their caliweights `caliweights = np.array(
        [0.388, 0.357, 0.038, 0.033, 0.017, 0.02, 0.016, 0.025, 0.002, 0.002, 0.002, 0.007, 0.006, 0.022, 0.004, 0.0004, 0.003, 0.002, 0.024, 0.029])`
The first is "Unannotated" = 0.388
And v2 [Changelog](http://www.scan-net.org/changelog#scannet-v2-2018-06-11)
> Updated annotations for all 1513 original ScanNet scans, with approx 90% vs previous 63% surface coverage.

**Comment by zaiweizhang** [comment]
No. The caliweights only apply to the 20 classes without "unannotated". It is an evaluation method proposed by the scannet paper in Table 7. (Paper: http://openaccess.thecvf.com/content_cvpr_2017/papers/Dai_ScanNet_Richly-Annotated_3D_CVPR_2017_paper.pdf)

**Comment by Bierfass** [comment]
Yes you are right. Thanks for the correction. 
But it still has to be ScanNet v1. If you compare the dates of the PointNet++ paper submit date (7 Jun 2017) and ScanNet v2 (2018-06-11) it is not possible that they are using v2

**Comment by zaiweizhang** [comment]
Yes. That's what I assumed. Just want to make sure.
